.github/workflows/ci.yaml

```diff
@@ -35,23 +35,9 @@ jobs:
       fail-fast: false
       matrix:
         # Test modern Odoo versions with latest Postgres version
-        odoo_version: ["17.0"]
-        pg_version: ["15"]
+        odoo_version: ["18.0"]
+        pg_version: ["16"]
         python_version: ["3.10"]
-        include:
```

Comment:
Why removing this?

Reply:
This is for testing v18 only. It is a temporal commit